.github/workflows/label_pr.yml

```diff
@@ -9,4 +9,5 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
+    - uses: actions/checkout@v3
```

Comment:
Seems extracted from #926

Reply:
ah yes, forget to checkout back to main when worked on this